components/post_view/combined_system_message/combined_system_message.jsx

```diff
@@ -3,11 +3,13 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-import {FormattedHTMLMessage, intlShape} from 'react-intl';
+import {intlShape} from 'react-intl';
 
-import {displayUsername} from 'mattermost-redux/utils/user_utils';
 import {Posts} from 'mattermost-redux/constants';
 
+import messageHtmlToComponent from 'utils/message_html_to_component';
+import {formatText} from 'utils/text_formatting.jsx';
```

Comment:
It seems convenient to use `Markdown` component but there's some issue on vertical spacing between group of system messages.